### ISSUE TITLE
Syntatic sugar for required are not rendered property

### ISSUE THREAD

**Issue by uudashr** [mulesoft/api-workbench#75]
Syntatic sugar not rendered properly

```raml
#%RAML 1.0
title: My API
version: v1
baseUri: http://api.my.com
types:
  Coordinate:
    type: object
    properties:
      latitude: number
      longitude: number
```

The longitude and latitude should be marked required on editor tools

**Comment by sichvoge** [comment]
Thanks for raising that. We will investigate.

**Comment by ddenisenko** [comment]
Fixed. Fix should be available in 0.8.10.
